examples/sql/sql_compact_example.py

```diff
@@ -1,7 +1,12 @@
 import dataclasses
 
 from hazelcast import HazelcastClient
-from hazelcast.serialization.api import CompactSerializer, CompactWriter, CompactReader
+from hazelcast.serialization.api import (
```

Comment:
Don't forget to change this file if you change `get_class` method's return type.

Reply:
At: https://github.com/hazelcast/hazelcast-python-client/pull/546/commits/6f6d8fdd8681b6349a84526961a55ef96e1a4eed